recipe/conda_build_config.yaml

```diff
@@ -24,7 +24,7 @@ cuda_compiler_version:
   - None
 
   - 9.2    # [linux64]
-  - 10.0   # [linux64 or win]
```

Comment:
Windows CUDA 10.0 is dropped here as it is incompatible with `vs2019`. Given there are already packages for it and conda-forge has largely dropped CUDA 10.0, we are skipping Windows CUDA 10.0 here